pymongo/operations.py

```diff
@@ -44,6 +45,35 @@
 _IndexKeyHint = Union[str, _IndexList]
 
 
+class _Operations(str, enum.Enum):
+    ABORT_TRANSACTION_OP = "abortTransaction"
```

Comment:
I'd optimzie for shorter names here. We can ditch the `_OP` suffix since these are all nested a named enum. We could also shorten `_Operations` to `_Op` so these become: `_Op.ABORT`